agent/structs/deep-copy.sh

```diff
@@ -21,10 +21,12 @@ deep-copy \
   -type GatewayService \
   -type GatewayServiceTLSConfig \
   -type HTTPHeaderModifiers \
+  -type HTTPRouteConfigEntry \
```

Comment:
Same thing as the generator code above - including the types here means they get `DeepCopy()` functions generated for them.

The generated code can be seen below in `agent/structs/structs.deepcopy.go`